### DISCUSSION TITLE
Unsigned Templates

### DISCUSSION THREAD

**Discussion by darkmac78** [projectdiscovery/nuclei-templates#10023] [General]
When starting a scan, I get a warning saying: [WRN] Loading 1 unsigned templates for scan. Use with caution.
Is it possible to see what is unsigned template?
I know I can use -dut to disable unsigned templates, I just wanna know if I can find out which is the unsigned one.

**Comment by rxerium** [comment]
Hey @darkmac78 the `-validate` flag may tell you which template is unsigned.
